### PR TITLE
Adds Label component #minor

### PR DESCRIPTION
**Pull request by Ditofry**
Sorry for the breach in protocol, I didn't notice the `CONTRIBUTING.md` for this until I had already written code. 
Fortunately this is a minuscule component, and if a "complete rewrite" is recommended it would probably take about 5 minutes.

This was the next available component in https://docs.google.com/spreadsheets/d/19o0dojwG5gx3vOnRf2XWKrE-AHfRAtz0GjrbKR0aVD8/edit#gid=0

It's a very simple interface:
`<Label className="extra-classname" text="green light" type="sent" />`

screen shot 2018-09-26 at 11 35 22 am

## Merge Checklist
- [x] QA'd by Jason Walker (@two24studios)
- [ ] Code reviewed by a frontend lead (Steve Kinney, Alex Wicks, Chris Chalstrom, Alex Thomsen)
- [x] If adding a new component, the component has been added to the exports in `src/index.ts`.
- [x] Workflow: Title has [semver](http://semver.org/) bump level defined (#major, #minor, #patch, or #noversion)

### CODE REVIEW

File: packages/ui-components/@types/styles/label.module.scss.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const label: string;
```
Comment (chrischalstrom) [inline packages/ui-components/@types/styles/label.module.scss.d.ts]: fyi you don't need to commit all this packages stuff

Comment (Ditofry) [inline packages/ui-components/@types/styles/label.module.scss.d.ts]: this is a new concept for me... a concept that I still don't really have.  Why do we need this?

File: src/label.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import Styles from './styles/label.module.scss';
+import cn from './utilities/classnames';
+
+export interface LabelProps {
+  className?: string;
+  text: string;
+  type: string;
+}
+
+export const Label: React.SFC<LabelProps> = ({ className, text, type }) => (
```
[the remark this answers is not in the document]
Comment (Ditofry) [reply src/label.tsx]: That's a really good idea

**Comment by two24studios** [comment]
Looks good to me from a style perspective, thanks @Ditofry !

**Pull request merged**